packages/expo-splash-screen/android/src/main/java/expo/modules/splashscreen/SplashScreenModule.kt

```diff
@@ -11,24 +10,21 @@ import expo.modules.kotlin.records.Record
 // Without this import any `SplashScreen.anyMethodName(...)` invocation on JS side ends up
 // in versioned SplashScreen kotlin object that stores no information about the ExperienceActivity.
 
-class HideSplashScreenOptions : Record {
-  @Field
-  val fade: Boolean = false
+class SplashScreenOptions : Record {
   @Field
   val duration: Long = 2000L
 }
 
 class SplashScreenModule : Module() {
-
   override fun definition() = ModuleDefinition {
     Name("ExpoSplashScreen")
 
-    AsyncFunction("hideAsync") { options: HideSplashScreenOptions? ->
-      SplashScreenManager.hide(options)
-    }.runOnQueue(Queues.MAIN)
+    Function("installSplashScreen") { options: SplashScreenOptions ->
```

Comment:
Instead of `installSplashScreen`,  I would go with `SplashScreen.setOptions` to avoid confusion. Because this is not really what install the splash screen, and even if the user is not calling this fn, the splash screen will appear.